src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -76,8 +76,8 @@
     public abstract String getFunctionName();
 
     /**
-     * Returns true if this step can accept implicit block as the argument.
-     *
+     * Return true if this step can accept an implicit block argument.
+     * (If it can, but it is called without a block, {@link StepContext#hasBody} will be false.)
```

Comment:
s/will/must|should/  given `hasBody` is the default implementation you can not know what subclasses will do?